From 90f87bfc5bbc26bc3bfeb5d89b89d5bd794b716b Mon Sep 17 00:00:00 2001
From: "M. Vefa Bicakci" <vefa.bicakci@windriver.com>
Date: Thu, 21 Apr 2022 02:06:11 +0000
Subject: [PATCH] Debian: Disable CONFIG_BNXT*

This commit disables the CONFIG_BNXT* options, because StarlingX ships
with an out-of-tree version of the bnxt_en kernel module. The
aforementioned configuration options enable the in-tree version of the
same module.

Signed-off-by: M. Vefa Bicakci <vefa.bicakci@windriver.com>
---
 debian/config/amd64/none/config | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/debian/config/amd64/none/config b/debian/config/amd64/none/config
index a5dadc7b6b55..5088a2aec7a2 100644
--- a/debian/config/amd64/none/config
+++ b/debian/config/amd64/none/config
@@ -2451,11 +2451,11 @@ CONFIG_TIGON3_HWMON=y
 CONFIG_BNX2X=m
 CONFIG_BNX2X_SRIOV=y
 # CONFIG_SYSTEMPORT is not set
-CONFIG_BNXT=m
-CONFIG_BNXT_SRIOV=y
-CONFIG_BNXT_FLOWER_OFFLOAD=y
-CONFIG_BNXT_DCB=y
-CONFIG_BNXT_HWMON=y
+# CONFIG_BNXT is not set
+# CONFIG_BNXT_SRIOV is not set
+# CONFIG_BNXT_FLOWER_OFFLOAD is not set
+# CONFIG_BNXT_DCB is not set
+# CONFIG_BNXT_HWMON is not set
 CONFIG_NET_VENDOR_BROCADE=y
 # CONFIG_BNA is not set
 CONFIG_NET_VENDOR_CADENCE=y
-- 
2.25.1

